dev-proxy-abstractions/IProxyConfiguration.cs

```diff
@@ -18,13 +18,18 @@ public enum LabelMode
 
 public interface IProxyConfiguration
 {
-    int Port { get; }
+    int ApiPort { get; }
```

Comment:
Sorted properties alphabetically to simplify lookup and management going forward